product_order_noname/models/product_template.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+    _order = "default_code, id"
```

Comment:
please use [new order](https://github.com/odoo/odoo/blob/15.0/addons/product/models/product_template.py#L19) criteria
    _order = "priority desc"

may be, I would add "default_code" here because if not the order will be only regarding the priority field

So, I would leave     _order = "priority desc, default_code"